src/selectors/per-thread/markers.js

```diff
@@ -372,6 +378,16 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
     MarkerTimingLogic.getMarkerTiming
   );
 
+  /**
+   * Creates the layout for the UserTiming markers so they can be displayed
+   * with the stack chart.
+   */
+  const getUserTimingMarkerTiming: Selector<MarkerTiming[]> = createSelector(
```

Comment:
It would be good to create a small test in `src/test/store/markers.test.js` that checks that this properly filters out the non-user timing markers. This function is super similar to `getNetworkTrackTiming`, which appears to have no test. I would recommend doing it like this...

```js
it('...', function() {
  const profile = getProfileWithMarkers([
    // Add user timing and non-user time markers.
  ]);
  const { getState } = storeWithProfile(profile);

  expect(
    selectedThreadSelectors.getUserTimingMarkerTiming(getState()),
  ).toEqual([
    // Copy/paste the output here.
  ]);
});
```